commands/local.js

```diff
@@ -40,6 +30,8 @@ module.exports.run = (args) => {
     if (err) {
       const parsedError = JSON.parse(err);
       console.log(parsedError);
+    } else if (Buffer.isBuffer(response)) {
+      process.stdout.write(response);
```

Comment:
Any way to test this? Could monkey path over `process.stdout.write` for the test? Is this any different that doing `console.log(response)`?

Reply:
`console.log` formats the output, so for writing binary data it doesn't work. `process.stdout.write` just dumps the output as is